lib/code_corps/accounts/accounts.ex

```diff
@@ -79,7 +102,7 @@ defmodule CodeCorps.Accounts do
       |> select([u], u.id)
       |> Repo.all
 
-    Task
+    CodeCorps.Task
```

Comment:
This is a little annoying, but may not be a problem depending on answer to the above.

Reply:
You can always do `alias X as: Y` to get around it. 

I do think we already have other places in code where we had to go by full module name due to this, though, so not a new thing here.